### ISSUE TITLE
Is slic3r compatible with SUNLU PLA filament?

### ISSUE THREAD

**Issue by huminlian** [alexrj/Slic3r#5164]
I own many SUNLU [PLA filament](https://www.sunlu.com/products/over-6kg-of-pla-pla-plus-3d-filaments-1kg-2-2lbs-fit-most-of-fdm-printer) spools. I usually use PrusaSlicer, so I don't know whether the slic3R can support it.

**Comment by rewolff** [comment]
yes (I don't have rights to this repo to close it. this is not an "issue" but a question, so can you close it?)
